.github/workflows/test.yml

```diff
@@ -6,15 +6,31 @@ on:
       - develop
   pull_request: {}
 
+concurrency: 
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
 jobs:
-  test:
+  test-and-bench:
     strategy:
       fail-fast: false
       matrix:
-        script: ["./test.sh", "./bench.sh"]
+        script: ["test", "bench"]
         os: ["ubuntu-latest", "macos-latest"]
+        wasmer-feature: ["wasmer_sys", "wasmer_wamr"]
+        exclude:
+          # TODO bench suite on macos-latest is killed by system due to running out of swap space
+          # All benches run fine individually
+          - script: bench
+            os: macos-latest
+            wasmer-feature: wasmer_wamr
     runs-on: ${{ matrix.os }}
     steps:
+      - name: increase swap space
+        if: matrix.os == 'ubuntu-latest'
+        uses: actionhippie/swap-space@v1
```

Comment:
In holochain there's an [action for ubuntu to free up disk space](https://github.com/holochain/holochain/blob/develop/.github/actions/common-pre/action.yml#L7) - does that achieve what you need the swap space for?

Reply:
No, the bench runs on CI were crashing when run in series due to running out of memory.